go.mod

```diff
@@ -1,30 +1,21 @@
 module github.com/flipkart-incubator/go-dmux
 
-go 1.12.4
+go 1.12
```

Comment:
probably bump the go version. 1.12 is very old.

Reply:
ack, changing it to 1.13.. due to some reason the 1.12.4 was giving error
" invalid go version '1.12.4': must match format 1.23"